src/Label.php

```diff
@@ -0,0 +1,375 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\WooCommercePriceLabels\Label.
+ */
+
+namespace Netzstrategen\WooCommercePriceLabels;
+
+use Endroid\QrCode\QrCode;
+
+/**
+ * Price label related functionality.
+ */
+class Label {
+
+  /**
+   * Paper size, orientation and font size for the price labels.
+   *
+   * @array
+   */
+  const PDF_LABEL_FORMATS = [
+    'A5|portrait|12px' => 'A5, portrait',
+    'A4|portrait|16px' => 'A4, portrait',
+    'A3|portrait|24px' => 'A3, portrait',
+    'A3|landscape|24px' => 'A3, landscape',
+  ];
+
+  /**
+   * Size of QR code to include in the PDF label document.
+   *
+   * @integer
+   */
+  const QR_CODE_SIZE_PORTRAIT = 400;
+  const QR_CODE_SIZE_LANDSCAPE = 600;
+
+  /**
+   * Generates a price label PDF document.
+   *
+   * @implements post_action_{$action}
+   */
+  public static function post_action_label($post_id) {
+    // Ensure user is allowed to print the price labels.
+    if (!current_user_can('print_price_label')) {
+      return;
+    }
+
+    $product = wc_get_product($post_id);
+
+    $regular_price = $product->get_regular_price();
+    $sale_price = $product->get_sale_price() ?: $regular_price;
+    if ($regular_price > $sale_price) {
+      $discount_percentage = round(100 * ($regular_price - $sale_price) / $regular_price);
+    }
+    else {
+      $discount_percentage = 0;
+    }
+
+    if (isset($_GET['format'])) {
+      $labelFormat = explode('|', $_GET['format']);
```

Comment:
To make it more clear, you should do something like: 
```php
// A5|portrait|12px
list($label_format, $orientation, $font_size) = explode('|', $_GET['format']);
```

Also I noticed: please don't mix naming conventions 🙃